instaloader/instaloadercontext.py

```diff
@@ -376,10 +376,13 @@ def get_json(self, path: str, params: Dict[str, Any], host: str = 'www.instagram
                 raise ConnectionException("HTTP error code {}.".format(resp.status_code))
             is_html_query = not is_graphql_query and not "__a" in params and host == "www.instagram.com"
             if is_html_query:
-                match = re.search(r'window\._sharedData = (.*);</script>', resp.text)
+                match = re.search(r"window\.__additionalDataLoaded\(\'/"+path+"\',(.*)\);</script>", resp.text)
```

Comment:
```suggestion
                match = re.search(r"window\.__additionalDataLoaded\(\'/"+path+r"\',(.*)\);</script>", resp.text)
```
(to make it pass the our pylint CI check)

Anyway, it looks to me semantically wrong to use a structure called **additional**DataLoaded in favor of the basic sharedData structure we use. Besides, it looks bad-practice to have the path dynamically inserted into the regular expression. What's the purpose of this, how does the structure look like, and which problem does it fix? Even tough it might eventually fix #380 (which I cannot reproduce at the moment), it looks like a solution that could break other parts, or cause data to be left out.

Reply:
The reason I've used the **additionalDataLoaded** is because I couldn't see in of the data needed to download all the pic of a multiple pics post while we are logged in in the **sharedData** structure.

Here's the answer I've got using instaloader : [https://gist.github.com/mseguin/7f109a641abcd2c7861f546216d17ac0](https://gist.github.com/mseguin/7f109a641abcd2c7861f546216d17ac0)

With the sharedData structure regex, you only get the following data which doesn't have the edge_sidecar_to_children data which is in the additionalDataLoaded.
```
window._sharedData = {"config":{"csrf_token":"ru45aMUjPiRnUpE3rrzbNICEvna0trGu","viewer":{"biography":"","external_url":null,"full_name":"Franck Desjardin","has_phone_number":false,"has_profile_pic":true,"id":"15417562916","is_joined_recently":false,"is_private":false,"profile_pic_url":"https://scontent-frt3-2.cdninstagram.com/vp/d03f770661905dc26455c8ae97a4ad8d/5E2AF16B/t51.2885-19/s150x150/70125060_370404283869985_4562914336909033472_n.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com","profile_pic_url_hd":"https://scontent-frt3-2.cdninstagram.com/vp/c80563568537ef248c57884fe8d4a56c/5E1E1E9B/t51.2885-19/s320x320/70125060_370404283869985_4562914336909033472_n.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com","username":"fdesjardin78","badge_count":"{\"seq_id\": 0, \"badge_count\": 0, \"badge_count_at_ms\": 1571069113264}"},"viewerId":"15417562916"},"country_code":"FR","language_code":"en","locale":"en_US","entry_data":{"PostPage":[{}]},"hostname":"www.instagram.com","deployment_stage":"c2","platform":"web","nonce":"cgzHfQrAnHlNhMyRyeb2gg==","mid_pct":30.72449,"zero_data":{},"cache_schema_version":3,"server_checks":{"hfe":true},"knobx":{"4":false,"17":false,"18":true,"19":true,"20":true},"to_cache":{"gatekeepers":{"4":true,"5":false,"6":false,"7":false,"8":false,"9":false,"10":false,"11":true,"12":false,"13":true,"14":true,"15":true,"16":false,"18":true,"19":false,"23":false,"24":false,"26":true,"27":false,"28":false,"29":true,"31":false,"32":true,"34":false,"35":false,"38":true,"40":true,"41":false,"43":false,"59":true,"60":false,"61":false,"62":false,"63":false,"64":false,"65":false,"66":true,"67":false,"68":false,"69":true,"70":false,"71":false},"qe":{"app_upsell":{"g":"","p":{}},"igl_app_upsell":{"g":"","p":{}},"notif":{"g":"","p":{}},"log_cont":{"g":"","p":{}},"onetaplogin":{"g":"","p":{}},"multireg_iter":{"g":"control_11_29","p":{"has_prioritized_phone":"false"}},"felix_clear_fb_cookie":{"g":"control","p":{"is_enabled":"true","blacklist":"fbsr_124024574287414"}},"felix_creation_duration_limits":{"g":"dogfooding","p":{"maximum_length_seconds":"3600","minimum_length_seconds":"60"}},"felix_creation_fb_crossposting":{"g":"control","p":{"is_enabled":"false"}},"felix_creation_fb_crossposting_v2":{"g":"control","p":{"is_enabled":"true","display_version":"2"}},"felix_creation_validation":{"g":"control","p":{"edit_video_controls":"true","description_maximum_length":"2200","max_video_size_in_bytes":"3600000000","minimum_length_for_feed_preview_seconds":"60","title_maximum_length":"75","valid_cover_mime_types":"image/jpeg,image/png","valid_video_extensions":"mp4,mov","valid_video_mime_types":"video/mp4,video/quicktime"}},"mweb_topical_explore":{"g":"","p":{}},"post_options":{"g":"control","p":{"use_refactor":"true","enable_igtv_embed":"true"}},"iglscioi":{"g":"","p":{}},"sticker_tray":{"g":"","p":{}},"web_sentry":{"g":"","p":{}},"0":{"p":{"4":true,"7":false,"8":false,"9":false},"qex":true},"2":{"p":{"0":true},"qex":true},"4":{"p":{"0":true},"qex":true},"5":{"p":{"1":false},"qex":true},"6":{"p":{"1":true,"5":false,"6":false,"7":false,"9":false,"10":false},"qex":true},"7":{"p":{},"qex":true},"10":{"p":{"2":false},"qex":true},"12":{"p":{"0":5},"qex":true},"13":{"p":{"0":true},"qex":true},"16":{"p":{"0":false},"qex":true},"17":{"p":{"1":true},"qex":true},"19":{"p":{"0":true},"qex":true},"21":{"p":{"2":false},"qex":true},"22":{"p":{"1":false,"2":8.0,"3":0.85,"4":0.95,"11":15,"12":3},"qex":true},"23":{"p":{"0":false,"1":false},"qex":true},"25":{"p":{},"qex":true},"26":{"p":{"0":""},"qex":true},"27":{"p":{"0":false},"qex":true},"28":{"p":{"0":false},"qex":true},"29":{"p":{},"qex":true},"30":{"p":{"0":true},"qex":true},"31":{"p":{},"qex":true},"33":{"p":{},"qex":true},"34":{"p":{"0":false},"qex":true},"36":{"p":{"0":true,"1":true,"2":false,"3":false,"4":false},"qex":true},"37":{"p":{"0":false},"qex":true},"39":{"p":{"0":false,"3":false,"4":false,"5":false,"6":false,"7":false},"qex":true},"40":{"p":{"0":false},"qex":true},"41":{"p":{"3":true},"qex":true},"42":{"p":{"0":true},"qex":true},"43":{"p":{"0":false,"1":false,"2":false},"qex":true},"44":{"p":{"1":"control","2":0.0},"qex":true},"45":{"p":{"0":false,"1":"not_now","2":false,"4":18,"5":false,"7":false,"8":2,"9":false,"10":"default","11":false,"12":false,"13":false,"14":false,"15":false,"16":18,"17":0,"18":false,"19":2},"qex":true},"46":{"p":{"0":false},"qex":true},"47":{"p":{"0":true,"1":false,"2":false,"3":false},"qex":true},"48":{"p":{"0":true},"qex":true},"49":{"p":{"0":false},"qex":true},"50":{"p":{"0":false},"qex":true},"51":{"p":{"0":true},"qex":true},"52":{"p":{"0":true},"qex":true},"53":{"p":{"0":30},"qex":true},"54":{"p":{"0":true},"qex":true},"55":{"p":{"0":false},"qex":true}},"probably_has_app":true,"cb":true},"device_id":null,"rollout_hash":"feea07f427eb","bundle_variant":"metro","is_canary":false};</script>
```

With the regex for the additionalDataLoaded you have the edge_sidecar_to_children information:
```
window.__additionalDataLoaded('/p/B3lEEu6Hgpl/',{"graphql":{"shortcode_media":{"__typename":"GraphSidecar","id":"2154145914197772901","shortcode":"B3lEEu6Hgpl","dimensions":{"height":1080,"width":1080},"gating_info":null,"fact_check_information":null,"media_preview":null,"display_url":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/4c94889a8ad9906d7f3674cf1195cced\/5E3E1C02\/t51.2885-15\/e35\/s1080x1080\/71210344_198640781156508_7847465905815217777_n.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com&_nc_cat=100","display_resources":[{"src":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/fae87a9e0d37771a56c6198c09288aaa\/5E1BD1B5\/t51.2885-15\/sh0.08\/e35\/s640x640\/71210344_198640781156508_7847465905815217777_n.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com&_nc_cat=100","config_width":640,"config_height":640},{"src":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/3800c6ba6b62285dfb2ed77e714de192\/5E6521B5\/t51.2885-15\/sh0.08\/e35\/s750x750\/71210344_198640781156508_7847465905815217777_n.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com&_nc_cat=100","config_width":750,"config_height":750},{"src":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/4c94889a8ad9906d7f3674cf1195cced\/5E3E1C02\/t51.2885-15\/e35\/s1080x1080\/71210344_198640781156508_7847465905815217777_n.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com&_nc_cat=100","config_width":1080,"config_height":1080}],"is_video":false,"tracking_token":"eyJ2ZXJzaW9uIjo1LCJwYXlsb2FkIjp7ImlzX2FuYWx5dGljc190cmFja2VkIjp0cnVlLCJ1dWlkIjoiOWY0NmJhNDRmM2JkNGVlNmFlMDg4N2MzNWJjYjE3ZWEyMTU0MTQ1OTE0MTk3NzcyOTAxIiwic2VydmVyX3Rva2VuIjoiMTU3MTA2OTExMzMyMnwyMTU0MTQ1OTE0MTk3NzcyOTAxfDE1NDE3NTYyOTE2fGNiMzRhNTQzMjE4YTM1ZWU1NTUwMDBmNmE4ZDRmNzMwY2VjODc2OTJkZDRiNDMxOTE4ZjNkMTIxOGFjZDFiYzYifSwic2lnbmF0dXJlIjoiIn0=","edge_media_to_tagged_user":{"edges":[{"node":{"user":{"full_name":"Anick Dumontet","id":"338019518","is_verified":false,"profile_pic_url":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/966ad72e746d4f08f0ae317fb8632bf4\/5E25EAFE\/t51.2885-19\/s150x150\/37057919_988728137974007_5077886262933716992_n.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com","username":"anickdumontet"},"x":0.1884444173,"y":0.6373332926}}]},"edge_media_to_caption":{"edges":[{"node":{"text":"Z\u00e9nitude en for\u00eat! \ud83e\uddd8\ud83c\udffb\u200d\u2640\ufe0f .\n.\n#sunday #automne #momentzen #walkintheforest #marcheenforet #parcmontstbruno"}}]},"caption_is_edited":false,"has_ranked_comments":true,"edge_media_to_parent_comment":{"count":22,"page_info":{"has_next_page":false,"end_cursor":null},"edges":[{"node":{"id":"17851740586621440","text":"wow tu est manifique \u2764\ufe0f\u2764\ufe0f","created_at":1571014730,"did_report_as_spam":false,"owner":{"id":"3589186471","is_verified":false,"profile_pic_url":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/743818b4210d78414435dee1661a8189\/5E1F09E2\/t51.2885-19\/13827206_272223633151757_377598655_a.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com","username":"danickaudet"},"viewer_has_liked":false,"edge_liked_by":{"count":1},"edge_threaded_comments":{"count":0,"page_info":{"has_next_page":false,"end_cursor":null},"edges":[]}}},{"node":{"id":"17844375316729297","text":"J'ai plusieurs souches sur ma terre \u00e0 bois o\u00f9 tu pourrais faire tes poses Yoga. :)","created_at":1571051567,"did_report_as_spam":false,"owner":{"id":"737745638","is_verified":false,"profile_pic_url":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/3a6202e0d439dab9410748d472604ff6\/5E236539\/t51.2885-19\/s150x150\/64613106_304449423843303_557599648236175360_n.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com","username":"pierre_depot"},"viewer_has_liked":false,"edge_liked_by":{"count":1},"edge_threaded_comments":{"count":0,"page_info":{"has_next_page":false,"end_cursor":null},"edges":[]}}},{"node":{"id":"17845885318716785","text":"Tr\u00e8s belle photo toujours aussi jolie bon week-end \u00e0","created_at":1571043637,"did_report_as_spam":false,"owner":{"id":"3923767023","is_verified":false,"profile_pic_url":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/72e55eff6294a5ad2a103539eab143e3\/5E2316E1\/t51.2885-19\/s150x150\/45695315_340650856488589_9201922832739598336_n.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com","username":"claudejamieson"},"viewer_has_liked":false,"edge_liked_by":{"count":1},"edge_threaded_comments":{"count":0,"page_info":{"has_next_page":false,"end_cursor":null},"edges":[]}}},{"node":{"id":"18069793399146749","text":"Tellement magnifique","created_at":1571026212,"did_report_as_spam":false,"owner":{"id":"4513051410","is_verified":false,"profile_pic_url":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/b118c2d935a893a5ec44de985ab3d686\/5E1FFE38\/t51.2885-19\/s150x150\/16110484_706914686152756_7933542917770051584_n.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com","username":"jnormand2011"},"viewer_has_liked":false,"edge_liked_by":{"count":1},"edge_threaded_comments":{"count":0,"page_info":{"has_next_page":false,"end_cursor":null},"edges":[]}}},{"node":{"id":"18076485100191912","text":"\ud83d\ude0d\ud83d\ude0d\ud83d\ude0d","created_at":1571024077,"did_report_as_spam":false,"owner":{"id":"5885903260","is_verified":false,"profile_pic_url":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/94225dcda0cf08407ff1ca1538c1cce1\/5E2D59E0\/t51.2885-19\/s150x150\/20837363_367297110354509_2433615829548400640_a.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com","username":"martin.boudrias.9"},"viewer_has_liked":false,"edge_liked_by":{"count":0},"edge_threaded_comments":{"count":0,"page_info":{"has_next_page":false,"end_cursor":null},"edges":[]}}},{"node":{"id":"17861459908530364","text":"Une tres belle  branche zen \ud83d\udd25","created_at":1571052994,"did_report_as_spam":false,"owner":{"id":"7830925799","is_verified":false,"profile_pic_url":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/8aa1e999b2bcc08df1e7ab250014296f\/5E24F2D0\/t51.2885-19\/s150x150\/69868009_447142882580592_3749807646874533888_n.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com","username":"gillo_123"},"viewer_has_liked":false,"edge_liked_by":{"count":1},"edge_threaded_comments":{"count":0,"page_info":{"has_next_page":false,"end_cursor":null},"edges":[]}}},{"node":{"id":"17875868557461194","text":"Une s\u00e9ance de karat\u00e9 kid \ud83e\udd70","created_at":1571055171,"did_report_as_spam":false,"owner":{"id":"1089573760","is_verified":false,"profile_pic_url":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/9385f15e68bf16802d1aa096b7b08eda\/5E28DDB4\/t51.2885-19\/s150x150\/40813700_286189622201763_8768766627794649088_n.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com","username":"sunnyboy200560"},"viewer_has_liked":false,"edge_liked_by":{"count":1},"edge_threaded_comments":{"count":1,"page_info":{"has_next_page":false,"end_cursor":null},"edges":[{"node":{"id":"17843466517731830","text":"@sunnyboy200560 \u2764\ufe0f","created_at":1571066009,"did_report_as_spam":false,"owner":{"id":"3987770453","is_verified":false,"profile_pic_url":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/ecb7ba0e0d0a7d45270d4a836654bfb6\/5E3ED935\/t51.2885-19\/s150x150\/18013287_282120882198571_1650785012523466752_a.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com","username":"toupin.alex"},"viewer_has_liked":false,"edge_liked_by":{"count":0}}}]}}},{"node":{"id":"17991536824275362","text":"\ud83d\udc4d\ud83d\ude0d","created_at":1571053208,"did_report_as_spam":false,"owner":{"id":"7862145372","is_verified":false,"profile_pic_url":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/1bf7d68a4bd2e69e9d9497d3f202d6ab\/5E23B167\/t51.2885-19\/s150x150\/32903490_176341406406930_6474599053383434240_n.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com","username":"yvonlacroix9"},"viewer_has_liked":false,"edge_liked_by":{"count":0},"edge_threaded_comments":{"count":0,"page_info":{"has_next_page":false,"end_cursor":null},"edges":[]}}},{"node":{"id":"17887060252426077","text":"Belle casquette de F1 \ud83c\udfce. Marcher dans la nature \u00e7a fait tellement du bien dans ce monde rempli de stress et souvent du stress inutile.","created_at":1571059070,"did_report_as_spam":false,"owner":{"id":"2350323152","is_verified":false,"profile_pic_url":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/4635c00f0399941a091d885dd5150900\/5E2B8489\/t51.2885-19\/s150x150\/12394120_1654478801460531_1768774102_a.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com","username":"robyjocelyn"},"viewer_has_liked":false,"edge_liked_by":{"count":1},"edge_threaded_comments":{"count":0,"page_info":{"has_next_page":false,"end_cursor":null},"edges":[]}}},{"node":{"id":"17882930359442644","text":"\u2764\ufe0f","created_at":1571060216,"did_report_as_spam":false,"owner":{"id":"7295209107","is_verified":false,"profile_pic_url":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/b50fd7d373c54f588dc27c62fbf81ed5\/5E1C8E09\/t51.2885-19\/s150x150\/29090296_1595551240561175_1721940989846749184_n.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com","username":"goulet8471"},"viewer_has_liked":false,"edge_liked_by":{"count":0},"edge_threaded_comments":{"count":0,"page_info":{"has_next_page":false,"end_cursor":null},"edges":[]}}},{"node":{"id":"17860485049557891","text":"Attention pour ne pas tomber Anick!!\ud83d\ude02","created_at":1571021445,"did_report_as_spam":false,"owner":{"id":"8321034764","is_verified":false,"profile_pic_url":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/ba2b82b08cf386fe40c81c1d6269716a\/5E341571\/t51.2885-19\/s150x150\/37193272_224849511551276_5199325818983022592_n.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com","username":"danyross6"},"viewer_has_liked":false,"edge_liked_by":{"count":1},"edge_threaded_comments":{"count":0,"page_info":{"has_next_page":false,"end_cursor":null},"edges":[]}}},{"node":{"id":"17943041311307706","text":"Anick tu va jouer dans karat\u00e9 girl\u2764\ufe0f","created_at":1571020306,"did_report_as_spam":false,"owner":{"id":"10675585447","is_verified":false,"profile_pic_url":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/49775666c35a9685dac8788a22768a5a\/5E338D68\/t51.2885-19\/s150x150\/55886222_267710784169477_4482478240929153024_n.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com","username":"yvon.michel.52"},"viewer_has_liked":false,"edge_liked_by":{"count":1},"edge_threaded_comments":{"count":0,"page_info":{"has_next_page":false,"end_cursor":null},"edges":[]}}},{"node":{"id":"17849654464641869","text":"Zen\ud83d\udc4d\ud83d\udc4d\ud83d\udc4d","created_at":1571017136,"did_report_as_spam":false,"owner":{"id":"3261826789","is_verified":false,"profile_pic_url":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/97094c7ca0e2423e0d7586d2e8868951\/5E2AB406\/t51.2885-19\/s150x150\/13734236_1662413814084776_761391792_a.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com","username":"biggy_bourgeois"},"viewer_has_liked":false,"edge_liked_by":{"count":1},"edge_threaded_comments":{"count":0,"page_info":{"has_next_page":false,"end_cursor":null},"edges":[]}}},{"node":{"id":"18005984182260905","text":"Tr\u00e8s bon pour tous \u2764\ufe0f","created_at":1571016965,"did_report_as_spam":false,"owner":{"id":"1539337312","is_verified":false,"profile_pic_url":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/2c0a9b9e298d6286f969b1261727946b\/5E30056B\/t51.2885-19\/s150x150\/44788222_1856603681055877_8359729395815940096_n.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com","username":"acuragl53"},"viewer_has_liked":false,"edge_liked_by":{"count":1},"edge_threaded_comments":{"count":0,"page_info":{"has_next_page":false,"end_cursor":null},"edges":[]}}},{"node":{"id":"17848142446665360","text":"C\u2019est \u00e7a \u00eatre en harmonie avec la nature \u00e7a fait du bien aux corps et \u00e0 l esprit","created_at":1571015143,"did_report_as_spam":false,"owner":{"id":"10865083177","is_verified":false,"profile_pic_url":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/8a831a62a2a09f7e04caef02daa036d5\/5E615705\/t51.2885-19\/s150x150\/50030096_2935964109759257_6929885747335921664_n.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com","username":"sylvain.picard.351"},"viewer_has_liked":false,"edge_liked_by":{"count":2},"edge_threaded_comments":{"count":0,"page_info":{"has_next_page":false,"end_cursor":null},"edges":[]}}},{"node":{"id":"18065230168160299","text":"Karat\u00e9 kid.. moment zen apr\u00e8s avoir fendu un arbre..\ud83d\ude06","created_at":1571020612,"did_report_as_spam":false,"owner":{"id":"8685374577","is_verified":false,"profile_pic_url":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/a6d3ef13c34a927e4f47d97e4d53e05d\/5E2BF903\/t51.2885-19\/s150x150\/70012517_506948943471838_3287135348634681344_n.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com","username":"juste1poff"},"viewer_has_liked":false,"edge_liked_by":{"count":1},"edge_threaded_comments":{"count":0,"page_info":{"has_next_page":false,"end_cursor":null},"edges":[]}}},{"node":{"id":"17909528782369551","text":"Magnifique","created_at":1571018070,"did_report_as_spam":false,"owner":{"id":"187312208","is_verified":false,"profile_pic_url":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/862be7ffb480c78080985fb89aac2654\/5E41FB15\/t51.2885-19\/s150x150\/22159015_1565606143521973_1187127109483495424_n.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com","username":"aceventura135"},"viewer_has_liked":false,"edge_liked_by":{"count":1},"edge_threaded_comments":{"count":0,"page_info":{"has_next_page":false,"end_cursor":null},"edges":[]}}},{"node":{"id":"18026717650224526","text":"Tu es zen et jolie","created_at":1571019230,"did_report_as_spam":false,"owner":{"id":"1621091849","is_verified":false,"profile_pic_url":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/436bc3ef77ad8337fddce85c750fcf00\/5E22CB38\/t51.2885-19\/s150x150\/23498501_1667003056645572_5601165360083501056_n.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com","username":"bonami1976"},"viewer_has_liked":false,"edge_liked_by":{"count":2},"edge_threaded_comments":{"count":0,"page_info":{"has_next_page":false,"end_cursor":null},"edges":[]}}},{"node":{"id":"17892501430408027","text":"\ud83d\udd25\ud83d\ude0d\ud83d\ude0d\u2764\ufe0fMY BEST XXX","created_at":1571018628,"did_report_as_spam":false,"owner":{"id":"4144657014","is_verified":false,"profile_pic_url":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/d33b990cb6db5538f951579bda5c01e6\/5E353BCF\/t51.2885-19\/s150x150\/19624881_1973756822909096_2236835748194025472_a.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com","username":"hearthb4u"},"viewer_has_liked":false,"edge_liked_by":{"count":1},"edge_threaded_comments":{"count":0,"page_info":{"has_next_page":false,"end_cursor":null},"edges":[]}}},{"node":{"id":"18107156869019757","text":"\u2764\ufe0fIl faut tellement en profiter \u2728","created_at":1571019326,"did_report_as_spam":false,"owner":{"id":"355859371","is_verified":false,"profile_pic_url":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/3e8c0aebd64f17160f5c46640b07371b\/5E3011E2\/t51.2885-19\/s150x150\/57355603_611516232691106_1459403932693430272_n.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com","username":"lestudiodeyogawanderlust"},"viewer_has_liked":false,"edge_liked_by":{"count":1},"edge_threaded_comments":{"count":0,"page_info":{"has_next_page":false,"end_cursor":null},"edges":[]}}},{"node":{"id":"17888499520414104","text":"\ud83c\udf1e\ud83c\udf1enice\ud83c\udf1e\ud83c\udf1e","created_at":1571062615,"did_report_as_spam":false,"owner":{"id":"3416800693","is_verified":false,"profile_pic_url":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/db735a62cdab62cfce0db27727ae9544\/5E1E2EAA\/t51.2885-19\/s150x150\/13329009_1611621022461486_109996138_a.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com","username":"yanickdouaire"},"viewer_has_liked":false,"edge_liked_by":{"count":0},"edge_threaded_comments":{"count":0,"page_info":{"has_next_page":false,"end_cursor":null},"edges":[]}}}]},"edge_media_preview_comment":{"count":22,"edges":[{"node":{"id":"17888499520414104","text":"\ud83c\udf1e\ud83c\udf1enice\ud83c\udf1e\ud83c\udf1e","created_at":1571062615,"did_report_as_spam":false,"owner":{"id":"3416800693","is_verified":false,"profile_pic_url":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/db735a62cdab62cfce0db27727ae9544\/5E1E2EAA\/t51.2885-19\/s150x150\/13329009_1611621022461486_109996138_a.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com","username":"yanickdouaire"},"viewer_has_liked":false,"edge_liked_by":{"count":0}}},{"node":{"id":"17843466517731830","text":"@sunnyboy200560 \u2764\ufe0f","created_at":1571066009,"did_report_as_spam":false,"owner":{"id":"3987770453","is_verified":false,"profile_pic_url":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/ecb7ba0e0d0a7d45270d4a836654bfb6\/5E3ED935\/t51.2885-19\/s150x150\/18013287_282120882198571_1650785012523466752_a.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com","username":"toupin.alex"},"viewer_has_liked":false,"edge_liked_by":{"count":0}}}]},"comments_disabled":false,"taken_at_timestamp":1571014225,"edge_media_preview_like":{"count":684,"edges":[{"node":{"id":"1856316","is_verified":true,"profile_pic_url":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/d002c1bf7f63f4b891e8cb1ba6f3e320\/5E623766\/t51.2885-19\/s150x150\/68928348_360831301538232_1261048255471419392_n.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com","username":"alexperron"}}]},"edge_media_to_sponsor_user":{"edges":[]},"location":{"id":"432056645","has_public_page":true,"name":"Mont Saint-Bruno","slug":"mont-saint-bruno","address_json":"{\"street_address\": \"\", \"zip_code\": \"\", \"city_name\": \"\", \"region_name\": \"\", \"country_code\": \"\", \"exact_city_match\": false, \"exact_region_match\": false, \"exact_country_match\": false}"},"viewer_has_liked":false,"viewer_has_saved":false,"viewer_has_saved_to_collection":false,"viewer_in_photo_of_you":false,"viewer_can_reshare":true,"owner":{"id":"338019518","is_verified":false,"profile_pic_url":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/966ad72e746d4f08f0ae317fb8632bf4\/5E25EAFE\/t51.2885-19\/s150x150\/37057919_988728137974007_5077886262933716992_n.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com","username":"anickdumontet","blocked_by_viewer":false,"followed_by_viewer":true,"full_name":"Anick Dumontet","has_blocked_viewer":false,"is_private":false,"is_unpublished":false,"requested_by_viewer":false},"is_ad":false,"edge_web_media_to_related_media":{"edges":[]},"edge_sidecar_to_children":{"edges":[{"node":{"__typename":"GraphImage","id":"2154145912872332024","shortcode":"B3lEEtrHWr4","dimensions":{"height":1080,"width":1080},"gating_info":null,"fact_check_information":null,"media_preview":"ACoqmlQCKTnkKDj2yRViJiu0g\/8ALNR\/KqshJjcnH3R\/6Ef15\/Kp43VUTPGEXn6kf59qzv2AivpZEAKngnkD8uvamwEbCD2P64Heo7i4jL4D5+gyM\/XFOA\/dtns3bGOw\/Om1oDEKfMGz0Bz9BipQT6t+lVWJ3gfw4OKvBBjp\/wCO1NiSJuMqf4l\/rUG4SRqh6bVGD9cZHp9fakdvlOOCRikiOz3OAPWmirDJbcRbFXkNzjjg\/Ucnj19KkjyibODufH48Y9vWnGUbhIeAmffn6D2z70izZVsjcS2c9M5x\/SqBokmBCr\/e3HjjuOKQXBHGf0NVZHDShsdDnHbirX2v\/YWpdhWIz5X95v8AvkY\/nSu6KrbWYkA44GP61VFWD\/x6yHvlRn2pmhQSQnIJ61Ye68pRgfMeBkDH1qM\/6rPfJ\/rUl0B9jiPcsc\/rQIjS9Y54Ge+Rn+dL9rk9azc81OAMUWA\/\/9k=","display_url":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/4c94889a8ad9906d7f3674cf1195cced\/5E3E1C02\/t51.2885-15\/e35\/s1080x1080\/71210344_198640781156508_7847465905815217777_n.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com&_nc_cat=100","display_resources":[{"src":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/fae87a9e0d37771a56c6198c09288aaa\/5E1BD1B5\/t51.2885-15\/sh0.08\/e35\/s640x640\/71210344_198640781156508_7847465905815217777_n.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com&_nc_cat=100","config_width":640,"config_height":640},{"src":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/3800c6ba6b62285dfb2ed77e714de192\/5E6521B5\/t51.2885-15\/sh0.08\/e35\/s750x750\/71210344_198640781156508_7847465905815217777_n.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com&_nc_cat=100","config_width":750,"config_height":750},{"src":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/4c94889a8ad9906d7f3674cf1195cced\/5E3E1C02\/t51.2885-15\/e35\/s1080x1080\/71210344_198640781156508_7847465905815217777_n.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com&_nc_cat=100","config_width":1080,"config_height":1080}],"accessibility_caption":"Image may contain: one or more people, tree, shoes, outdoor and nature","is_video":false,"tracking_token":"eyJ2ZXJzaW9uIjo1LCJwYXlsb2FkIjp7ImlzX2FuYWx5dGljc190cmFja2VkIjp0cnVlLCJ1dWlkIjoiOWY0NmJhNDRmM2JkNGVlNmFlMDg4N2MzNWJjYjE3ZWEyMTU0MTQ1OTEyODcyMzMyMDI0Iiwic2VydmVyX3Rva2VuIjoiMTU3MTA2OTExMzM2MHwyMTU0MTQ1OTEyODcyMzMyMDI0fDE1NDE3NTYyOTE2fDZiNzVlOTZmMjE1YTk4MGUyNjkyN2QwM2UyYjZhNWRiNmViMDBlMmY4YmU5MjE5Zjg0MmRlNGQzNWVjYTllODIifSwic2lnbmF0dXJlIjoiIn0=","edge_media_to_tagged_user":{"edges":[{"node":{"user":{"full_name":"Anick Dumontet","id":"338019518","is_verified":false,"profile_pic_url":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/966ad72e746d4f08f0ae317fb8632bf4\/5E25EAFE\/t51.2885-19\/s150x150\/37057919_988728137974007_5077886262933716992_n.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com","username":"anickdumontet"},"x":0.1884444173,"y":0.6373332926}}]}}},{"node":{"__typename":"GraphImage","id":"2154145912880658431","shortcode":"B3lEEtrnHf_","dimensions":{"height":1078,"width":1080},"gating_info":null,"fact_check_information":null,"media_preview":"ACoprM+1sKSN239GNXGYlTnAz3qg7Dd1zx+u6pGf5D9DzUWJLluu7Gf8+9JecFff5ePqDT7IjA7cdu9RXmCB7OP1pdQLKsfNyR\/Cf50pZM9P8\/lVVpf3vp8o\/nT\/ADh60DMpYmzgjB9\/rU8MIdWHHUgjJGceh9T70LnOTx9asfKvIIznPXrVop6EUb7F2rkD3PSmPuYYyQM7vxp5JycdM1KsDvxg89M8UhlJnfdn8KZ5jev6Vfe3YcEcj3FQGP2oCxYSLPBAHfJ5\/wA\/lVOEGOY56AkH8jWnD\/n9apQf61v+Bfyq0Sy6t0MYAyfoP61A905PBx7Z7fWoW\/rUZ7VIx7OSckkk\/wCeabsNTt1X\/d\/pWe3U\/WkO5\/\/Z","display_url":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/db76492d9e299b0534730763dd0675ac\/5E635243\/t51.2885-15\/e35\/s1080x1080\/73290821_137044970969121_3077235085037404056_n.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com&_nc_cat=109","display_resources":[{"src":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/e9d46edc5ed649dee7776faeea258e62\/5E3129F4\/t51.2885-15\/sh0.08\/e35\/s640x640\/73290821_137044970969121_3077235085037404056_n.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com&_nc_cat=109","config_width":640,"config_height":639},{"src":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/f66ac1461ac735b01be27d9ed4005ffd\/5E63FBF4\/t51.2885-15\/sh0.08\/e35\/s750x750\/73290821_137044970969121_3077235085037404056_n.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com&_nc_cat=109","config_width":750,"config_height":748},{"src":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/db76492d9e299b0534730763dd0675ac\/5E635243\/t51.2885-15\/e35\/s1080x1080\/73290821_137044970969121_3077235085037404056_n.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com&_nc_cat=109","config_width":1080,"config_height":1078}],"accessibility_caption":"Image may contain: one or more people, people standing, tree, plant, outdoor and nature","is_video":false,"tracking_token":"eyJ2ZXJzaW9uIjo1LCJwYXlsb2FkIjp7ImlzX2FuYWx5dGljc190cmFja2VkIjp0cnVlLCJ1dWlkIjoiOWY0NmJhNDRmM2JkNGVlNmFlMDg4N2MzNWJjYjE3ZWEyMTU0MTQ1OTEyODgwNjU4NDMxIiwic2VydmVyX3Rva2VuIjoiMTU3MTA2OTExMzM2MHwyMTU0MTQ1OTEyODgwNjU4NDMxfDE1NDE3NTYyOTE2fDMyMzEwMjkxNGFjZmZkYWE3MDdhMDViY2E5NmNkZWI3OTEwNWQyNDFjYTA0Y2ZiYmNjMzZiNzNjZDMzMjllYWQifSwic2lnbmF0dXJlIjoiIn0=","edge_media_to_tagged_user":{"edges":[{"node":{"user":{"full_name":"Anick Dumontet","id":"338019518","is_verified":false,"profile_pic_url":"https:\/\/scontent-frt3-2.cdninstagram.com\/vp\/966ad72e746d4f08f0ae317fb8632bf4\/5E25EAFE\/t51.2885-19\/s150x150\/37057919_988728137974007_5077886262933716992_n.jpg?_nc_ht=scontent-frt3-2.cdninstagram.com","username":"anickdumontet"},"x":0.231111084,"y":0.627555542}}]}}}]}}}});</script>
```

I guess this solution could break something else but I didn't occur any error for the moment using it. If you see a better solution, I am more than open to learn an other way to do it, but a least for the moment I can use Instaloader while I still have the #380 bug with Instaloader 4.2.9.